llvm.sh

```diff
@@ -170,8 +171,20 @@ if [[ "${VERSION_CODENAME}" == "bookworm" ]]; then
     # https://github.com/llvm/llvm-project/issues/62475
     add-apt-repository -y "${REPO_NAME}"
 fi
+if [[ "${VERSION_CODENAME}" == "trixie" ]]; then
```

Comment:
Thanks for the change! 

Could you please use a list/array instead?
I think it is going to grow.